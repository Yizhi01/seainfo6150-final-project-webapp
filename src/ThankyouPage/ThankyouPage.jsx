import React from "react";
import { Link } from "react-router-dom";
import styles from "./ThankyouPage.module.css";

const ThankyouPage = () => {
  return (
    <div className={styles.thankyouContainer}>
      <div className={styles.tyBackground}>
        <div className={styles.contentArea}>
          <h1>Thank You!</h1>
          <p>Your recipe have been sent to our team, we will review it and reach out to you soon!</p>
          <Link to="/">Back to Home</Link>
        </div>
      </div>
    </div>
  );
};

export default ThankyouPage;