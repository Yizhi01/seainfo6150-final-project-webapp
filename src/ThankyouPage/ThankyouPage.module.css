.thankyouContainer {
  position: relative;
  justify-content: center;
  text-align: center;
  min-height: 100vh;
  color: #fff;
}

.tyBackground {
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  background: url("../images/thankyou-large.jpg") no-repeat center center fixed;
  background-size: cover;
  overflow: visible;
}

.contentArea {
  height: 100%;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
}

.thankyouContainer h1{
  font: bold 60px "open Sans", sans-serif;
  margin-bottom: 15px;
}

.thankyouContainer p{
  font: normal 28px "open Sans", sans-serif;
  margin: 0 100px 40px;
}

.contentArea a {
  padding: 20px 46px;
  background: none;
  color: #fff;
  border: 2px solid #fff;
  text-decoration: none;
}
