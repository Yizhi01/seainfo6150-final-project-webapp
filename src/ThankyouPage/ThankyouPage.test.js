import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter as Router } from "react-router-dom";
import ThankyouPage from "./ThankyouPage.jsx";

it("renders the ThankyouPage component correctly", () => {
  const { container } = render(
    <Router>
        <ThankyouPage />
    </Router>
  );
  expect(container).toMatchSnapshot();
});