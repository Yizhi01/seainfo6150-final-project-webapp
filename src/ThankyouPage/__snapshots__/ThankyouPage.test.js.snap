// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the ThankyouPage component correctly 1`] = `
<div>
  <div
    class="thankyouContainer"
  >
    <div
      class="tyBackground"
    >
      <div
        class="contentArea"
      >
        <h1>
          Thank You!
        </h1>
        <p>
          Your recipe have been sent to our team, we will review it and reach out to you soon!
        </p>
        <a
          href="/"
        >
          Back to Home
        </a>
      </div>
    </div>
  </div>
</div>
`;
